README.md

```diff
@@ -15,11 +15,12 @@ You need to have the KVM hypervisor installed, along with a few other packages:
 - libguestfs-tools-c
 - qemu-img
 - libvirt-client
+- libosinfo-bin
```

Comment:
I think this should just be `libosinfo` without the `-bin`. What OS are you using?